core/vm/contracts.go

```diff
@@ -93,6 +95,8 @@ var PrecompiledContractsBerlin = map[common.Address]PrecompiledContract{
 	common.BytesToAddress([]byte{7}): &bn256ScalarMulIstanbul{},
 	common.BytesToAddress([]byte{8}): &bn256PairingIstanbul{},
 	common.BytesToAddress([]byte{9}): &blake2F{},
+
+	common.BytesToAddress([]byte{100}): &voteSignatureVerify{},
```

Comment:
This is also a hardfork change, you can not simply put it inside `PrecompiledContractsBerlin`

Reply:
Fixed